_posts/2019-08-12-how-to-configure-production-grade-aws-account-structure.adoc

```diff
@@ -2566,139 +2644,50 @@ include {
-      pgp_key              = "keybase:bob"
-      create_login_profile = true
-      create_access_keys   = false
-    }
-    alice = {
-      groups               = ["billing"]
-      pgp_key              = "keybase:carol"
-      create_login_profile = true
-      create_access_keys   = false
-    }
-  }
+[source,json]
+----
+{
+  "UserId": "AIDAXXXXXXXXXXXX:1597932316055520000",
+  "Account": "<STAGE_ACCOUNT_ID>",
+  "Arn": "arn:aws:sts::<STAGE_ACCOUNT_ID>:assumed-role/OrganizationAccountAccessRole/1597932316055520000"
 }
 ----
 
-The changes we've made are:
-
-. **Enable CloudTrail**. We've set `enable_cloudtrail = true` and configured CloudTrail to use the S3 bucket and KMS
-  CMK in the logs account. Note that we can only do this now, after creating the logs account and applying a security
-  baseline to it.
-
-. **Enable AWS Config**. We've set `enable_config = true` and configured AWS Config to use the S3 bucket in the logs
-  account. Note that we can only do this now, after creating the logs account and applying a security
-  baseline to it.
-
-. **Enable AWS Config Rules**. On each of the child accounts in `child_accounts`, we've set `enable_config_rules = true`
-  to enable AWS Config Rules in that account. Note that we can only do this now, after creating each child account and
-  applying a security baseline to it.
-
-Authenticate once more to the root account and run `terragrunt apply -parallelism=4`:
+You're now ready to deploy the `account-baseline` module in the stage account by running `terragrunt apply`:
 
 [source,bash]
 ----
-cd infrastructure-live/root/_global/account-baseline
-terragrunt apply -parallelism=4
+cd infrastructure-live/stage/_global/account-baseline
+aws-vault exec stage-from-root -- terragrunt apply -parallelism=2
 ----
 
-IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=4`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors. On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid `Error: pipe: too many open files` errors by running: `ulimit -n 1024`.
+IMPORTANT: We **strongly** recommend setting Terraform parallelism to a low value (e.g., `-parallelism=2`) with the `account-baseline-xxx` modules. This is because these modules deploy multi-region resources (e.g., GuardDuty, AWS Config, etc), and for each region, Terraform spins up a separate process, so if you don't limit the parallelism, it may peg all your CPU cores and lead to network connectivity errors. On some operating systems, such as MacOS, you may also need to increase your open files limit to avoid `Error: pipe: too many open files` errors by running: `ulimit -n 1024`.
```

Comment:
Same comment as above re:`ulimit`.

Reply:
Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/1d9b5b79c84044d7f8fb6225df4379a2399bdfe8 and https://github.com/gruntwork-io/gruntwork-io.github.io/pull/354/commits/c6a37ff8c87fd62b63173a56e3b286cff2f81b8b.